webClient/src/app/core/menu-bar/menu-bar.config.ts

```diff
@@ -173,8 +176,9 @@ export const MENU = [
                 action: {
                     internalName: 'saveFile'
                 },
-                keyMap: '' // [Ctrl+S] won't work due to browser conflicts. Keybindings will need
-            },              // to be rethinked.
+                keyMap: '', // [Ctrl+S] won't work due to browser conflicts. Keybindings will need
```

Comment:
Is this still relevant. Does Ctrl + S conflict with things? Can we just be brave and use Alt + S, is it worth it? @NakulManchanda @1000TurquoisePogs

Reply:
Eh, Ctrl + S is too common, I don't even like this suggestion.